tests/cast/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -32,4 +36,8 @@ $bar = (unset) $foo;
 $bar = (int) $foo;
 $bar = (int) $foo;
 
+$binary = (string) $string;
+$binary = b'binary string';
+$binary = b'binary string';
```

Comment:
@czosel yes, because after `b` is string node

Reply:
But for regular strings we're just re-printing quotes as we find them, right?

Reply:
@czosel yes

Reply:
Alright, doesn't that mean that we should do the same here? Otherwise, cases with embedded quotes might break?
```php
b"test'quote'"
```

Reply:
@czosel i add more tests, looks like we don't break code

Reply:
Alright - I just don't like the inconsistency too much. Before, I didn't look at the code yet, and now I saw that it just behaves like this because the parser returns `isDoubleQuote: false` for `b"foo"` - that's a bug, right?

Reply:
@czosel yep, it is bug in parser, my solution is workaround

Reply:
Alright, can you report an issue in the parser? Then I'm fine with merging this as is :smile:

Reply:
@czosel Already reported https://github.com/glayzzle/php-parser/issues/147

Reply:
To me, that is a different error. I reported the error I meant here: https://github.com/glayzzle/php-parser/issues/149